src/prettytable/prettytable.py

```diff
@@ -448,6 +487,16 @@ def _validate_attributes(self, name, val):
     # ATTRIBUTE MANAGEMENT       #
     ##############################
 
+    @property
+    def xhtml(self):
+        """Print <br/> tags if True, <br> tags if false"""
+        return self._xhtml
+
+    @xhtml.setter
+    def xhtml(self, val):
+        self._validate_option("xhtml", val)
+        self._xhtml = val
```

Comment:
Are these out of scope of this PR? Would be nice to keep it a bit tighter, we can make a new PR if needed.

Reply:
I see your point - please note that if I split this to a new PR, my PR will be depended on it.

I needed the xhtml property because of the change I made in `_get_options()` method - I omitted the "_" from the `getattr` call.

Please let me know what do you prefer

Reply:
Let's remove it from this one, thanks!

Reply:
I made a new PR with this change - #115 
Once it will be merged, I'll be able to remove this change.